server/controllers/entities/lib/get_wikidata_enriched_entities.coffee

```diff
@@ -45,10 +45,11 @@ getEnrichedEntity = (wdId)->
 
 mergeWdAndInvData = (entity, invEntity)->
   if entity.missing? then return formatEmpty 'missing', entity
-  { P31 } = entity.claims
-  if P31
-    simplifiedP31 = wdk.simplifyPropertyClaims P31
-    entity.type = getEntityType simplifiedP31.map(prefixify)
+  { P31, P279 } = entity.claims
+  if P31? or P279?
```

Comment:
Thats also technical debt, no ? what if 3 other property claims are added next time ?
Cant we use an object or array to store claims and a function to simplify and assign type to entity directly

Reply:
[P31 and P279 are the 2 properties explicitly recommended by the Wikidata community to handle typing](https://www.wikidata.org/wiki/Wikidata:Item_classification#Classification_in_Wikidata), and as such requested to be defined, at least one or the other, on each Wikidata entity. Meanwhile, on Inventaire, each entity has a P31 claim and no P279

Reply:
[P31 and P279 are the 2 properties explicitly recommended by the Wikidata community to handle typing](https://www.wikidata.org/wiki/Wikidata:Item_classification#Classification_in_Wikidata), and as such requested to be defined, at least one or the other, on each Wikidata entity. Meanwhile, on Inventaire, each entity has a P31 claim and no P279